media-libs/smpeg/smpeg-0.4.5.recipe

```diff
@@ -7,9 +7,9 @@ HOMEPAGE="http://www.icculus.org/smpeg"
 COPYRIGHT="1999-2004 Sam Lantinga, Joe Tennies."
 LICENSE="GNU LGPL v2"
 REVISION="8"
-SOURCE_URI="https://github.com/icculus/smpeg/archive/refs/tags/release-$portVersion.tar.gz"
+SOURCE_URI="https://github.com/icculus/smpeg/archive/refs/tags/release_0_4_5.tar.gz"
```

Comment:
For the next time: you could have used `${portVersion//./_}` here. This replaces the `.` in $portVersion with `_`.

Reply:
Already put that down on my wiki, but as this isn't developed anymore I just left it as is :)

https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#portversion